### DISCUSSION TITLE
How to deposit while I am in Pakistan 🇵🇰

### DISCUSSION THREAD

**Discussion by Harrysworldpk** [se4/huobi-all-in#3] [General]
Kindly give me the best suggestions

**Comment by 6r6** [comment]
Hi Bro,

I suggest using binance OTC service, to deposit. Then transfer to Huobi with `TRC20` Chain ( Very small consumption, roughly 1 USD ) if you want to transfer your assets to Huobi.

**Comment by 6r6** [comment]
A friendly reminder that this project is very outdated and use at your own risk :) 😄
